src/services/event.ts

```diff
@@ -496,15 +502,17 @@ export async function findAllEvents(): Promise<EventShape[]> {
 const splitPipe = (str: string) => str.split('\n').map((s) => s.trim()).filter(Boolean);
 
 const mappings: Record<string, string> = {
-  Audience: 'audience',
-  Creator: 'creator',
-  'Edit Title': 'eventName',
+  Audience: 'eventIntendedAudience',
```

Comment:
with the changes to the targets of the remapping, I think this JIRA needs to include a migration file to remap the current existing objects to match the new names/structure.

Reply:
All the changed targets are corrections to fields that were named one way in this file and one way in the form; that is to say, eventIntendedAudience & eventDuration are already populated in existing reports. This change simply correctly imports them to begin with.

Reply:
but the data is in the blob under the old name for the old records, right?

Reply:
The historical blobs have correct values under the correct (what we've updated to here) names.